src/UuidFactory.php

```diff
@@ -180,6 +187,24 @@ public function setUuidBuilder(UuidBuilderInterface $builder)
         $this->uuidBuilder = $builder;
     }
 
+    /**
+     * @return ValidatorInterface
+     */
+    public function getValidator()
+    {
+        return $this->validator;
+    }
+
+    /**
+     * @return void
+     */
+    public function setValidator(ValidatorInterface $validator)
+    {
+        assert(func_num_args() === 1);
```

Comment:
Not sure this is really necessary. If you have less than one arg, the call will fail as $validator cannot be null, and if there are more than one, well they'll just be ignored.